Начинаем программировать

HardLaif opened the pull request:
Сделал домашку

---
:mortar_board: [Начинаем программировать](https://up.htmlacademy.ru/react-individual/3/user/2385327/tasks/4)

HardLaif committed:
2.16. Начинаем программировать

Сделал домашку

KletskovG commented on src/App.tsx:
```diff
@@ -0,0 +1,9 @@
+import '../markup/css/main.css';
```
1) Стоит вынести файл из директорию markup в src, внутри markup просто шаблонный код, который тебе предоставили готовым

В идеале эту директорию можно удалить

2) Импорт файлов стилями всегда пишут последним в списке импортов, такая практика 

Пример 

```tsx
import { Spin } from '@components/Spin';

import './SportCalendar.scss';
```

KletskovG commented on src/App.tsx:
```diff
@@ -0,0 +1,9 @@
+
+export default function App(){
+  return (
+    <MainPage></MainPage>
```
```tsx
<MainPage />
```

В JSX так же можно писать теги без закрывающего, если нет детей

KletskovG commented on src/components/EmptyFavorites/EmptyFavorites.tsx:
```diff
@@ -0,0 +1,51 @@
+export default function EmptyFavorites(){
```
Не стоит делать две отдельных директории под одну и ту же страницу в разных состояниях

лучше сделать 

src/components/Favorites/Favorites.tsx
src/components/Favorites/EmptyFavorites.tsx

KletskovG replied:
Все файлы должны называться в lower-case через дефис, такие критерии у Академии, хотя твой вариант UpperCamelCase лучше

KletskovG replied:
Такой же момент с разбиением страниц на директории нужно сделать и в остальных местах

HardLaif committed:
Сделал исправления

css импортировал из папки public, перемещать никуда не стал, думаю это не сильно критично.
названия файлов поменял на кебаб,
последовательность импортов поправил, сделал закрывашки для компонентов без дочерних элементов
